docs/topics/kotlin-doc.md

```diff
@@ -137,36 +135,10 @@ Names in links are resolved using the same rules as if the name was used inside
 In particular, this means that if you have imported a name into the current file, you don't need to fully qualify it
 when you use it in a KDoc comment.
 
-Note that KDoc does not have any syntax for resolving overloaded members in links. Since the Kotlin documentation generation
+Note that KDoc does not have any syntax for resolving overloaded members in links. Since Kotlin's documentation generation
 tool puts the documentation for all overloads of a function on the same page, identifying a specific overloaded function
 is not required for the link to work.
 
-## Module and package documentation
```

Comment:
Oof, I don't think this part is actually covered in Dokka's documentation, it might instead link to this section, actually. And I'm not 100% sure it should migrate there

The primary concern is that this format might already be used by other tools and thus relied upon. I'm afraid we'd be rug pulling if Dokka became its owner and started changing things for its own use.

The safest option would be to re-write this section to be tool-independent, i.e just describe what it could be used for and what the format is, and then mention at the end that Dokka can understand it. 

But since there's no predefined name/path for such files (unlike with Java's `package-info.java`), I don't know how useful it'll be and whether we should keep it here

I'll ask for a second opinion and come back to you

Reply:
We've discussed it with the team and came to the conclusion that it's unlikely that anyone other than Dokka is using it, and it's unlikely that this will be part of the KDoc specification itself in its current state.

So we've agreed to move it to Dokka's documentation. I can't think of a place where we can put it though, since it's runner-independent, so it looks like it's going to be a separate page altogether

@sarahhaggarty do you have any objections with this? I can move it by the time we deploy documentation live, but you can do it yourself if you have the time

Reply:
@IgnatBeresnev, no objections from me. Do you know if we have an example of this I could preview somewhere?